### DISCUSSION TITLE
How to find wall height.

### DISCUSSION THREAD

**Discussion by AfxentiouNikolas** [IfcOpenShell/IfcOpenShell#3028] [Q&A] [answered]
Hello,
I would like to inquire about obtaining the height for each wall using ifc parser. As a newcomer to the parser, I am struggling to find a solution to this problem. My objective is to determine the dimensions (length, width, and height) of all eight walls in my simple house, designed using revit, in order to calculate the building's area and volume.

Currently, I have successfully identified the total number of walls and their respective material names, as well as the length and width of each wall. However, I have encountered difficulty in determining the height of each wall.

Would you be kind enough to assist me with this issue?

Thank you for your time and consideration.

Sincerely,
Nikolas

![Capture](https://user-images.githubusercontent.com/97607951/233037076-fcb8e07d-4da0-4f0e-9763-9da9390f7839.PNG)

**Comment by arunkvijay** [comment, marked as answer]
You can use  `ifcopenshell.util.element.get_psets(wall, qtos_only=True)` if you are using the high level ifcopenshell python api

**Reply by AfxentiouNikolas** [reply]
Thank you so much for your reply!
